### DISCUSSION TITLE
Add a marker pen tool to create highlights

### DISCUSSION THREAD

**Discussion by claudio4** [go-shiori/shiori#626] [Feature requests]
I would like to request a feature for Shiori that I miss from Pocket: the ability to highlight specific parts of an article. In my opinion, having this tool in Shiori would greatly enhance its suitability for long-term bookmark management. It would allow users to not only save the entire text but also mark the portions they find most meaningful. It would be even better if Shiori could also provide the option to attach notes to these highlights.

**Comment by Wittionary** [comment]
Yes, this is exactly the purpose for which I read - to learn. I learn by making brief notes and comments on designated sentences or paragraphs.
